locale/en.json

```diff
@@ -151,6 +155,7 @@
 		"identity-block.reset-password-submit-submitted": "Continue to log in",
 		"identity-block.save": "Save",
 		"identity-block.sign-up": "Sign Up",
+		"identity-block.sign-up-natural": "Sign up",
```

Comment:
These new and changed keys will need to be added to our Lokalise account, has that been done yet?

Reply:
Unfortunately, we don't have access to the themes lokalise account.